install_config/topics/ocp_osp_provisioning.adoc

```diff
@@ -175,6 +175,14 @@ openshift_node_open_ports:
   port: 53/tcp
 - service: dns udp
   port: 53/udp
+
+# To set the service network CIDR, uncomment the following property and set its value.
+# The value must be smaller than the CIDR value set in the inventory file as
+# openshift_openstack_kuryr_pod_subnet_cidr. By default, this value is /24.
```

Comment:
you are missing the line to uncomment, e.g:
openshift_openstack_kuryr_pod_subnet_cidr: 24

Reply:
:open_mouth:  Thanks!

Reply:
https://github.com/openshift/openshift-docs/pull/21998/commits/119744c47994ba25604705f71f85ac5483bb9263